### PR TITLE
adjust the panel content color

### PR DESCRIPTION
**Pull request by nabim777**
## Description

In this PR there is working for the panel content to adjust the color consistent with the `nextcloud` panel
image

## Related Issue or Workpackage

- https://community.openproject.org/projects/nextcloud-integration/work_packages/52979/activity

## Screenshots (if appropriate):

After adjusting the UI look like this
### dark mode
image

### light mode
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [x] Updated `CHANGELOG.md` file

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Replaced vue loading icon with default Nextcloud loading icon [#561](https://github.com/nextcloud/integration_openproject/pull/561)
 - Fixes: Signing terms and services for special user "OpenProject" when `terms_of_service` app is enabled [#552](https://github.com/nextcloud/integration_openproject/pull/552)
 - Fix error log when fetching non-existent work package [#573](https://github.com/nextcloud/integration_openproject/pull/573)
```
Comment (SagarGi) [inline CHANGELOG.md:10]:
```suggestion
```

**Review by SagarGi — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (individual-it) [reply CHANGELOG.md:10]: Please do that in a separate PR, every PR should only contain changes that relate to the description

**Review by individual-it — COMMENTED** [review]
please also test with the high-contrast themes

File: CHANGELOG.md (hunk 1 above)
Comment (SagarGi) [reply CHANGELOG.md:10]: sure will create one!

**Comment by nabim777** [comment]
> please also test with the high-contrast themes

I think it works fine in high-contrast themes
## High contrast  mode
image

## Dark theme with high contrast mode
image

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
...
+- Adjust the `Open project` panel's content color to match the `Upcoming events` panel's content. [#583](https://github.com/nextcloud/integration_openproject/pull/583)
```
Comment (individual-it) [inline CHANGELOG.md:11]: I've just learned that we don't put in the links in the changelogs

**Review by individual-it — CHANGES_REQUESTED** [review]

**Comment by SagarGi** [comment]
@nabim777  https://github.com/nextcloud/integration_openproject/pull/584 This PR has been merged please rebase it and push.

File: CHANGELOG.md (hunk 2 above)
Comment (individual-it) [reply CHANGELOG.md:11]: So we need to do extra work when preparing the release. If the link is not needed I would suggest to remove it always, so when releasing we only need to change the header

Comment (SagarGi) [reply CHANGELOG.md:11]: Yeah that can be also done.

**Review by SagarGi — APPROVED** [review]
LGTM :+1:

**Comment by nabim777** [comment]
new workpackages are created to remove the `!` character from the dashboard content  [github-PR-link](https://github.com/nextcloud/integration_openproject/pull/590)

**Review by individual-it — APPROVED** [review]

**Pull request merged**